fixups/FileRedirectionFixup/PathRedirection.cpp

```diff
@@ -244,19 +250,35 @@ void Log(const wchar_t* fmt, ...)
             va_start(args2, fmt);
             count = std::vswprintf(wstr.data(), wstr.size() + 1, fmt, args2);
             va_end(args2);
-            assert(count >= 0);
         }
         wstr.resize(count);
 #if _DEBUG
         ::OutputDebugStringW(wstr.c_str());
-		::OutputDebugStringW(L"\n");
+		//::OutputDebugStringW(L"\n");

```

Comment:
Did you mean to leave this commented out?  If we don't need it can it be removed.

Reply:
removed.